package/Shaders/Lighting.hlsl

```diff
@@ -512,6 +512,13 @@ Texture2D<float4> TexLandNormal4Sampler : register(t10);
 Texture2D<float4> TexLandNormal5Sampler : register(t11);
 Texture2D<float4> TexLandNormal6Sampler : register(t12);
 
+Texture2D<float4> TexLandTHDisp0Sampler : register(t92);
```

Comment:
Does this need to be behind a define? What happens if we disable your feature?

Reply:
If my feature is disabled this is undefined, but that should be okay because the only place it is called in extendedmaterials is used only when some extended bits are set, which by default they are all 0 even if my feature is unloaded.